one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+THis file was also added in the feature1 branch
+
+Another change in the feature-1 branch
```

Comment:
It is good that you have decided to add this line of code

Reply:
Thanks for the comment Ed